### PR TITLE
Resolve #3: Ensure Consistent DB Initialization and Correct Migrations for DEV environment

### LINKED ISSUE #3: Fix Inconsistent DB Initialization and Faulty Migrations in Dev Environment
#### Problem
Our project's development environment faces two main issues: inconsistent database initialization and faulty migration scripts. This inconsistency leads to a challenging setup for developers, impacting productivity and hindering new feature development.

#### Expected Behavior
- Easy and consistent database initialization for all developers.
- Accurate and up-to-date database migrations reflecting the current schema.
- A reliable method to reset and update the development database.

#### Current Issues
- Database initialization varies across environments, causing setup failures.
- Migrations do not match the actual desired schema, leading to errors.
- Lack of a straightforward process to reset the development database.

#### Steps to Reproduce
1. Follow the setup instructions to initialize the development database.
2. Run migration scripts and encounter errors or schema inconsistencies.
3. Try to reset the development database with no clear or reliable method available.

#### Proposed Solution
- Audit and correct existing migrations for consistency.
- Introduce a script or tool for reliable database initialization and resetting.
- Document the process for database setup and management in the development environment.

Your input and help in resolving these issues would greatly improve our development workflow.

### PR DESCRIPTION
**Pull request by artemkozak**

### CODE REVIEW

**Comment by conradogarciaberrotaran** [comment]
added pre-commit, fixes pytest path issue.

File: backend/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ poetry install
 3. Setup the database:
 
 ```shell
-setup/setup_db.py
+setup/build.sh
```
Comment (artemkozak) [inline backend/README.md]: Rollback to setup_db.sh
Reason: unclear name 'build', it's hard to understand what this script will build (eg docker container, house, etc.)

File: backend/pyproject.toml [hunk 2]
```diff
@@ -32,4 +36,26 @@ build-backend = "poetry.core.masonry.api"
...
     ".venv/*",
     "alembic/*",
-]
\ No newline at end of file
+]
+
+[tool.ruff]
+src = ["app", "tests"]
+line-length = 88 # same as flake8
+ignore = [
```
Comment (artemkozak) [inline backend/pyproject.toml:43]: @conradogarciaberrotaran Does it make sence to move this to a separate config file (eg .flake8)?

File: backend/pyproject.toml [hunk 3]
```diff
@@ -32,4 +36,26 @@ build-backend = "poetry.core.masonry.api"
 omit = [
     ".venv/*",
     "alembic/*",
-]
\ No newline at end of file
+]
+
+[tool.ruff]
+src = ["app", "tests"]
+line-length = 88 # same as flake8
```
Comment (artemkozak) [inline backend/pyproject.toml]: Let's set it 80 or 120.
88 is too weird.

File: backend/README.md (hunk 1 above)
Comment (artemkozak) [reply backend/README.md]: Moved pre-commit initialization to README.md

File: backend/pyproject.toml (hunk 2 above)
Comment (artemkozak) [reply backend/pyproject.toml:43]: In the latest version of Python, it is better to store these settings in the pyproject.toml file.

File: backend/pyproject.toml (hunk 3 above)
Comment (artemkozak) [reply backend/pyproject.toml]: The line length was set to 80

**Review by conradogarciaberrotaran — APPROVED** [review]

**Pull request merged**